csrankings.csv

```diff
@@ -5330,7 +5332,6 @@ Ernst-Rüdiger Olderog,University of Oldenburg,https://uol.de/en/computingscienc
 Erol Sahin,Middle East Technical University,http://www.kovan.ceng.metu.edu.tr/~erol/Home.html,aO-849gAAAAJ
 Errol L. Lloyd,University of Delaware,https://www.eecis.udel.edu/~elloyd,805n5zYAAAAJ
 Ertan Onur,Middle East Technical University,https://eonur.ceng.metu.edu.tr/blog,6ybYW3QAAAAJ
-Erwin P. Bottinger,Hasso Plattner Institute,https://hpi.de/en/the-hpi/people/professors/prof-dr-ewin-boettinger.html,3dWDnnYAAAAJ

```

Comment:
This is required. See the DBLP entry. https://dblp.org/pers/b/Bottinger:Erwin_P=.html

Reply:
I realized that there is another same entry for Prof. Bottinger in the file, and deleted the duplicated one.

Reply:
It's not duplicated. It's a different entry without an umlaut. It needs to be there. Look at the DBLP page and you will see *two* names.